scripts/merge_repos_and_check_duplicates.py

```
@@ -1,97 +1,118 @@
 def get_config_variable(config_name, variable, delimiter):
-    config = open(config_name, 'r')
-    
-    # initialize empty list to store ignored files
     ignored_files = []
-    
-    for i in config:
-        if search(variable, i):
-            # transform row with ignored files into list of variables
-            ignored_files = (i.split('='))[1].strip('\n').strip('"').strip("'").split(delimiter)
-    
+    with open(config_name, "r") as config:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)